kernel/src/device/tty/mod.rs

```diff
@@ -31,9 +32,48 @@ pub use device::TtyDevice;
 pub use driver::{PushCharError, TtyDriver};
 pub(super) use n_tty::init;
 pub use n_tty::{iter_n_tty, system_console};
+pub(crate) type Result<T> = core::result::Result<T, Error>;
```

Comment:
Isn't this already available due to `use prelude::*`? Why do we need to define it again?